### PR TITLE
Synch beta doc updates to master. Closes #962.

### LINKED ISSUE #962: Synch latest beta doc updates with master
## Specify type:
- Enhancement

## Description:
Synch latest beta doc updates with master

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: docs/source/configure.rst [hunk 1]
```diff
@@ -80,24 +80,24 @@ The ``qpc server config`` command takes the following options:
...
+  # qpc server config --host 192.0.2.0 --port 8443
 
 .. _login:
 
-Logging in to and Logging out of the qpc Command Line Interface
----------------------------------------------------------------
-After the connection between the qpc command line tool and the Quipcords server is configured on the system where you want to use the qpc command line interface, you can log in to the interface and begin using it to run qpc commands.
+Logging in to the Quipucords Server
+-----------------------------------
+After the connection between the qpc command line tool and the Quipucords server is configured on the system where you want to use the command line interface, you can log into the server and begin using the command line interface to run qpc commands.
```
Comment (abaiken) [inline docs/source/configure.rst]: Grammar is not my strong point but we use both `log in to` and `log into` --> not sure if we want to change this to be consistent or if it doesn't make a difference.

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -2,18 +2,18 @@ Getting Started with Quipucords
...
 
 You can save multiple credentials and sources to use with Quipucords in various combinations as you run inspection processes, or *scans*. When you have completed a scan, you can access the collection of *facts* in the output as a *report* to review the results.
 
 Before You Begin: Check the Connection to the Quipucords Server
 ---------------------------------------------------------------
-In some organizations, a single person might be responsible for scanning IT resources. However, in others, multiple people might hold this responsibility. Any additional Quipucords users who did not install the Quipucords server and command line tool must ensure that their command line tool instance is configured to connect to the server and that they can log in to the command line interface.
+In some organizations, a single person might be responsible for scanning IT resources. However, in others, multiple people might hold this responsibility. Any additional Quipucords users who did not install the Quipucords server and command line tool must ensure that the command line tool instance is configured to connect to the server and that the user name can log in to the command line interface.
```
Comment (abaiken) [inline docs/source/getting_started.rst]: Maybe change "and that the user name can log in" ---> to something like "and that additional users have log in capabilities." Right now the wording sounds strange.

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -192,14 +196,19 @@ When you stop a scan by using the ``scan pause`` command, you can restart that s
...
-
-  ::
+3. When you are ready to start the scan job again, enter the command to restart the scan. For example, to restart scan ``1``, you would enter the following command::
 
     # qpc scan restart --id 1
+
+
+Logging out of the Quipucords Server
+------------------------------------
+When you log in to the server, the command retrieves a token that is used for authentication with subsequent command line interface commands. That token expires daily. In addition, the token is removed when you log out of the server.
```
Comment (abaiken) [inline docs/source/getting_started.rst]: whatever we decide about "into" vs. "in to" should be carried over to here as well.

File: docs/source/install.rst [hunk 4]
```diff
@@ -113,65 +111,70 @@ When your system meets the minimum required kernel release, you can use the foll
...
 
-  #  sudo docker load -i quipucords.1.0.0.tar.gz
-  ...
-  Loaded image: quipucords:1.0.0
+3. Verify the image within the local Docker registry by entering the following command::
 
+    #  sudo docker images
 
-You can verify the image within the local Docker registry::
+The output appears similar to the following example::
```
Comment (abaiken) [inline docs/source/install.rst]: Earlier in the page, we say "the output of this command is similar to the following example::" (line 104) Do we want to have consistent language, or is it okay that it varies a little?

File: docs/source/configure.rst (hunk 1 above)
Comment (chambridge) [reply docs/source/configure.rst]: These are Jamie's edits that I'm porting forward so I leave the word smithing to her.

File: docs/source/getting_started.rst (hunk 2 above)
Comment (chambridge) [reply docs/source/getting_started.rst]: These are Jamie's edits that I'm porting forward so I leave the word-smithing to her.

File: docs/source/getting_started.rst (hunk 3 above)
Comment (chambridge) [reply docs/source/getting_started.rst]: yeah i don't know the rules. Let Jamie do Jamie.

File: docs/source/install.rst (hunk 4 above)
Comment (chambridge) [reply docs/source/install.rst]: That is a good note. I'll try and get this updated in the future.

**Pull request merged**